### PR TITLE
Chore: Exit process on `unhandledRejection` on CI

### PR DESCRIPTION
**Pull request by sampaiodiego**

### CODE REVIEW

File: apps/meteor/app/voip/server/startup.ts [hunk 1]
```diff
@@ -14,6 +14,8 @@ settings.changeMultiple(
 	(_values) => {
 		// Here, if 4 settings are changed at once, we're getting 4 diff callbacks. The good part is that all callbacks are fired almost instantly
 		// So to avoid stopping/starting voip too often, we debounce the call and restart 1 second after the last setting has reached us.
-		return debouncedRefresh();
+		if (settings.get('VoIP_Enabled')) {
+			return debouncedRefresh();
+		}
```
Comment (ggazzo) [inline apps/meteor/app/voip/server/startup.ts:19]: changeMultiple is already debounced, so there is no need for the line 6

Comment (ggazzo) [reply apps/meteor/app/voip/server/startup.ts:19]: image

Comment (ggazzo) [reply apps/meteor/app/voip/server/startup.ts:19]: image

Comment (sampaiodiego) [reply apps/meteor/app/voip/server/startup.ts]: the debounce was higher than the settings' default (1000ms while settings' is 500ms), but I agree 500ms should be enough, so removed.

this comment from your image was not being used as documentation for editor's intellisense, so I changed that as well

**Comment by sampaiodiego** [comment]
btw, with the fixes introduced by this PR I think we should make `test-ee` CI step mandatory to merge.

**Review by ggazzo — APPROVED** [review]

File: apps/meteor/app/voip/server/startup.ts (hunk 1 above)
Comment (ggazzo) [reply apps/meteor/app/voip/server/startup.ts]: image
its possible to change the default if 1s is the exactly value

**Pull request merged**